### DISCUSSION TITLE
When is it worth spidering if there are no coordinates?

### DISCUSSION THREAD

**Discussion by arrival-spring** [alltheplaces/alltheplaces#10107] [Q&A]
I've found a few brands which I would want to spider but do not have coordinates available, maybe they just have a list of stores with the address, but not even a Google Maps link.

Obviously we want coordinates when possible but then e.g. mcdonalds_za has a spider which has no coordinates as nothing seems to be available.

What would the threshold be for when it's worth it? An address? A phone number? A website? A ref/id of some kind?

**Comment by Cj-Malone** [comment]
Decent addresses can be geocoded. Without coords or addresses it's hard to use in a map context. but it can still be used in a list format eg https://www.theukhighstreet.com/p/about

**Comment by CloCkWeRX** [comment]
I think the question becomes "how many locations?"

5? No coords? Ugh, high effort, low value.
100? Different story.

You might want to strike a balance of putting a direct link to the storefinder into wikidata if avail; as a `location info url`
That provides the trade off of "I want to know where I can find a brand X store" being answerable as "This is the list of locations"; for tools that use / consume that.

**Comment by arrival-spring** [comment]
Thanks for the answers, that makes sense. I'll balance hard work vs amount of data available
